packages/webui/src/components/Menu.test.tsx

```diff
@@ -201,6 +201,30 @@ describe('Menu', () => {
       expect(fn3).not.toHaveBeenCalled();
     });
 
+    it('should include event object in callback when clicked', async () => {
+      const { getByRole, getAllByTestId } = render(<Menu label="Menu" items={itemsWithCallback} />);
+      const menu = getByRole('menu');
+
+      await act(async () => {
+        await userEvent.click(menu);
+      });
+
+      await act(async () => {
+        const user = userEvent.setup();
+
+        const listItems = getAllByTestId('menu-items-item');
+        const firstItem = listItems.at(0)!;
+
+        await user.keyboard('{Shift>}');
+        await user.keyboard('{Meta>}');
+        await user.click(firstItem);
+        await user.keyboard('{/Shift}');
+        await user.keyboard('{/Meta}');
```

Comment:
💬 In the `userEvent` library, `{Key>}` means holding the key down, and `{/Key}` means releasing it 🤷.  Basically these five lines perform a click whilst holding (and then releasing) "Shift" and "Win/Cmd" keys.